holoviews/plotting/bokeh/renderer.py

```diff
@@ -108,20 +108,3 @@ def get_size(self_or_cls, plot):
         Returns a tuple of (width, height) in pixels.
         """
         return (plot.state.height, plot.state.height)
-
-
-    @classmethod
-    def validate(cls, options):
-        """
-        Validates a dictionary of options set on the backend.
-        """
-        try:
-            import bokeh
-            import bokeh.io
-        except:
-            raise ImportError("Could not import one of bokeh, pandas or scipy.")
-
```

Comment:
The baseclass implements the basics of validate, and if you can't import bokeh you wouldn't have gotten as far as calling BokehRenderer.validate anyway.